src/services/commit/useCompareTotals.tsx

```diff
@@ -14,10 +14,11 @@ import {
   RepoOwnerNotActivatedErrorSchema,
 } from 'services/repo/schemas'
 import Api from 'shared/api'
+import { type NetworkErrorObject } from 'shared/api/helpers'
 import A from 'ui/A'
 
 const CoverageObjSchema = z.object({
-  percentCovered: z.number().nullable(),
+  coverage: z.number().nullable(),
```

Comment:
On the api side it seems like these fields (`percentCovered`, `coverage`) return the same data and `percentCovered` is preferred. Are we seeing data returned with one but not the other? Because that would be a good thing for us to fix if so

https://github.com/codecov/codecov-api/blob/324fd728cd905c073e96c6657ac3d363a92c6070/graphql_api/types/coverage_totals/coverage_totals.py#L5

https://github.com/codecov/codecov-api/blob/324fd728cd905c073e96c6657ac3d363a92c6070/graphql_api/types/coverage_totals/coverage_totals.graphql#L10

Reply:
`coverage` is deprecated, however because that would require a lot of FE changes, the simple solution that was implemented was to alias the fields.

```graphql
coverage: Float @deprecated(reason: "Use `percentCovered`")
```

Reply:
Got it, thanks! I missed that there was a mismatch somewhere in the flow causing this problem.